io.catenax.packing_list/1.0.0/PackingList.ttl

```diff
@@ -0,0 +1,155 @@
+@prefix ext-number: <urn:samm:io.catenax.shared.business_partner_number:1.0.0#>.
+@prefix ext-uuid: <urn:samm:io.catenax.shared.uuid:1.0.0#>.
+
+:PackingList a samm:Aspect;
+    samm:preferredName "Packing List"@en;
+    samm:description "This model represents a commonly used packing list used in logistics. It is linked to the logistics twin. "@en;
+    samm:properties (:orderDate :shippingDate :listItems :senderAddress :receiverAddress :orderNumber);
+    samm:operations ();
+    samm:events ().
+:orderDate a samm:Property;
+    samm:preferredName "Order Date"@en;
+    samm:description "Date once the item(s) were ordered. "@en;
+    samm:characteristic :DateCharacteristic;
+    samm:exampleValue "2023-01-12"^^xsd:date.
+:shippingDate a samm:Property;
+    samm:preferredName "Shipping Date"@en;
+    samm:description "Date once item(s) are shipped. "@en;
+    samm:characteristic :DateCharacteristic;
+    samm:exampleValue "2023-02-11"^^xsd:date.
+:listItems a samm:Property;
+    samm:preferredName "List Items"@en;
+    samm:description "Set of items part of the packing list. "@en;
+    samm:characteristic :SetOfListItems.
+:senderAddress a samm:Property;
+    samm:preferredName "Sender Address"@en;
+    samm:description "Sender of the delivery, including the name and address. "@en;
+    samm:characteristic :SenderAddressCharacteristic.
+:receiverAddress a samm:Property;
+    samm:preferredName "Receiver Address"@en;
+    samm:description "Receiver of the delivery, including the name and address. "@en;
+    samm:characteristic :ReceiverAddressCharacteristic.
+:DateCharacteristic a samm:Characteristic;
+    samm:preferredName "DateCharacteristic"@en;
+    samm:dataType xsd:date.
+:SetOfListItems a samm-c:Set;
+    samm:preferredName "Set Of List Items"@en;
+    samm:description "Set Of List Items of the packing list. "@en;
+    samm:dataType :ListData.
+:ListData a samm:Entity;
+    samm:preferredName "List Data"@en;
+    samm:description "Catena-X ID and data of the packing list item."@en;
+    samm:properties (:quantity :position [
+  samm:property :itemDescription;
+  samm:optional "true"^^xsd:boolean
+] :itemName [
+  samm:property :catenaXId;
+  samm:optional "true"^^xsd:boolean
+]).
+:quantity a samm:Property;
+    samm:preferredName "Quantity"@en;
+    samm:description "Quantity of the item. "@en;
+    samm:characteristic :QuantityCharacteristic.
+:position a samm:Property;
+    samm:preferredName "Position"@en;
+    samm:description "Numbering of all positoins or items in the packing list. "@en;
+    samm:characteristic :PositionCharacteristic;
+    samm:exampleValue "1"^^xsd:int.
+:itemDescription a samm:Property;
+    samm:preferredName "Item Description"@en;
+    samm:description "Describes the item in the packing list. "@en;
+    samm:characteristic samm-c:Text.
+:itemName a samm:Property;
+    samm:preferredName "Item Name"@en;
+    samm:description "Item Name from a packing list. Should it be a serial number (CatenaX-ID?)"@en;
```

Comment:
dot at the end. Questionmark in the description.

Reply:
fixed